xbmc/settings/lib/SettingDefinitions.h

```diff
@@ -64,6 +64,7 @@
 #define SETTING_XML_ATTR_SETTING      "setting"
 #define SETTING_XML_ATTR_BEFORE       "before"
 #define SETTING_XML_ATTR_AFTER        "after"
+#define SETTING_XML_ATTR_HIDE         "hide"
```

Comment:
This isn't a very obvious name as it could be interpreted as "hide the whole group".
Also I'm not sure why you need two options. What different cases do you want to cover?